crates/hyle-modules/src/node_state/api.rs

```diff
@@ -81,7 +81,17 @@ pub async fn get_contract(
 ) -> Result<impl IntoResponse, AppError> {
     let name_clone = name.clone();
     match state.bus.shutdown_aware_request::<()>(name).await {
-        Ok(contract) => Ok(Json(contract)),
+        Ok((block_height, contract)) => Ok(Json(ApiContractData {
+            contract_name: name_clone,
+            state_block_height: block_height,
+            state_commitment: contract.state,
+            program_id: contract.program_id,
+            verifier: contract.verifier,
+            timeout_window: match contract.timeout_window {
+                TimeoutWindow::NoTimeout => None,
+                TimeoutWindow::Timeout(window) => Some(window.0),
+            },
+        })),
```

Comment:
this is breaking change for most of the apps, if you merge this, you need to update all nodes & cargo update all apps & provers